xfstests/mount.fuse.passthrough

```diff
@@ -2,16 +2,40 @@
 
 ulimit -n 1048576
 
-# It would be easier if we could just set SCRATCH_DEV and TEST_DEV to the source directory
-# path in local.options. Unfortunately, setting these variables to a path seems get
-# xfstests all worked up (even though it should treat these as opaque values), and it
-# refuses to even start running any tests).
 dev="$1"
 shift
-if [ "$dev" = "${SCRATCH_DEV}" ]; then
-  source="${SCRATCH_SOURCE}"
-else
-  source="${TEST_SOURCE}"
+mnt="$1"
+shift
+# -o
+shift
+mntopts="$1"
+shift
+
+# source can be provided as NFS style device (e.g. TEST_DEV=source:/${TEST_SOURCE})
+# or by exporting the {TEST,SCRATCH}_SOURCE env vars
+# and/or it can already be inside mount options (passthrough_ll style)
+if ( echo "$mntopts" | grep -q "source=" ) ; then
+	# Don't pass source as position argument
+	source=
+elif [[ "$dev" == "source:"* ]]; then
+	source="${dev#"source:"}"
+elif [ "$dev" = "${SCRATCH_DEV}" ]; then
+	source="${SCRATCH_SOURCE}"
+elif [ "$dev" = "${TEST_DEV}" ]; then
+	source="${TEST_SOURCE}"
```

Comment:
@bsbernd Please remove the references to `*_SOURCE` here as well.

Reply:
Thanks, I was already in the process to do that. Can you please check the new  additional patch?